interval/window.go

```diff
@@ -0,0 +1,310 @@
+package interval
+
+import (
+	"github.com/influxdata/flux/codes"
+	"github.com/influxdata/flux/internal/errors"
+	"github.com/influxdata/flux/values"
+)
+
+// TODO(nathanielc): Make the epoch a parameter to the window
+// See https://github.com/influxdata/flux/issues/2093
+const epoch = values.Time(0)
```

Comment:
It seems like calling this concept "epoch" could be a little confusing, since its variable and not the standard epoch that's used in programming.

Reply:
@jsternberg I've moved the todo to the Window struct. I agree that it makes more sense there. As far as allowing epoch to be more customizable, I think we might be able to wait on those changes until we actually start working on the issue to add epoch to windows fully